pkg/operators/v1alpha1/catalogsource_types.go

```diff
@@ -86,6 +92,28 @@ type CatalogSourceSpec struct {
 	Icon        Icon   `json:"icon,omitempty"`
 }
 
+// GrpcPodConfig contains configuration specified for a catalog source
+type GrpcPodConfig struct {
+	// NodeSelector is a selector which must be true for the pod to fit on a node.
+	// Selector which must match a node's labels for the pod to be scheduled on that node.
+	// More info: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+	// +optional
+	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
+
+	// Tolerations are the catalog source's pod's tolerations.
+	// +optional
+	Tolerations []corev1.Toleration `json:"tolerations,omitempty"`
+
+	// If specified, indicates the pod's priority. "system-node-critical" and
```

Comment:
nit: This comment feels ripe for rot to me. Do we need to call out the PriorityClass implementation details as they exist today?

Reply:
Great call out! I've updated!